features/steps/steps.py

```diff
@@ -485,6 +511,44 @@ def step_impl(context, representation_id):
     
     handle_errors(context, errors)
 
+@then("The value must {constraint}")
+@then("The values must {constraint}")
+@then('At least "{num:d}" value must {constraint}')
+@then('At least "{num:d}" values must {constraint}')
+def step_impl(context, constraint, num=None):
+    errors = []
+    
+    within_model = getattr(context, 'within_model', False)
+
+    if constraint.startswith('be ') or constraint.startswith('in '):
+        constraint = constraint[3:]
+
+    if constraint.startswith('in ') or constraint.startswith('in '):
+        constraint = constraint[3:]
+
+
+    if getattr(context, 'applicable', True):
+        stack_tree = list(filter(None, list(map(lambda layer: layer.get('instances'), context._stack))))
+        instances = [context.instances] if within_model else context.instances
+
+        if constraint[-5:] == ".csv'":
+            csv_name = constraint.strip("'")
+            for i, values in enumerate(instances):
+                if not values:
+                    continue
+                attribute = getattr(context, 'attribute', None)
+
+                dirname = os.path.dirname(__file__)
+                filename = Path(dirname).parent / f"resources/{csv_name}"
```

Comment:
This is a bit inconsistent. If you use Path() with the / operator then do it consistently:

```suggestion
                filename = Path(dirname).parent / "resources" / csv_name
```

Reply:
Ah, I did not see this was possible, thanks.